dedicated-postgres-clusters.mdx

```diff
@@ -0,0 +1,123 @@
+
+![Postgres branches allocated to shared and dedicated clusters](images/branches-clusters.png)
+
+The key features of dedicated clusters are:
+
+- Xata databases and branches can be allocated to a dedicated cluster. You can allocate a single branch (for example, only your production branch) or multiple branches (for example, all your development branches).
+- You'll be able to move Postgres databases between dedicated clusters with no or minimal downtime.
+- You can configure dedicated clusters with a configurable number of read replicas, and you can ask for them to be in different availability zones.
+- You can choose between a wide range of [instance types](https://xata.io/docs/dedicated-cluster#provisioned-instances) or you can define a minimum and maximum CPU/RAM configuration and let [auto-scaling](https://xata.io/docs/dedicated-cluster#auto-scaling-instances) choose the right size dynamically based on load. Scaling up is almost instant and scaling down is done automatically in a matter of minutes.
+- The per-GB cost of storage is significantly cheaper (20 times less!) than the storage cost for shared clusters.
+- The dedicated clusters are free from limitations we impose on our shared clusters to protect from abuse: rate and concurrency limits as well as some limitations on the SQL commands you can execute. You can see a comparison table in the [docs](https://xata.io/docs/postgres#supported-statements).
+
+Before we dive into the specifics of dedicated clusters, let's take a step back and provide some insight into our current architecture so you can understand the difference and the importance of having optionality.
+
+## Shared cell architecture
+
+Xata has a **cell-based architecture**, in which each cell consists of a Postgres Aurora cluster, an Elasticsearch cluster, a queue, and the associated services that serve the Data API, the SQL wire protocol, and more. The cells form our data plane where we serve our users through our Data API and the Postgres wire protocol.
+
+We have several cells in one region and each of them is multi-tenant. Essentially, they are being shared between several users, which allows us to offer a generous [free tier](https://xata.io/blog/serverless-postgres-platform#the-free-tier) (15GB of storage, HA included, no pausing) and in general a very good offering for “small scale”. Having this cell-splitting system in a region ensures that one noisy neighbor will not affect the whole region. It will, however, affect the cell they live in. In order to mitigate the noisy neighbor issues, we have rate limits and concurrency limits in place and we are monitoring for abuse.
```

Comment:
```suggestion
We have several cells in one region and each of them is multi-tenant. Essentially, they are being shared between several users, which allows us to offer a generous [free tier](/blog/serverless-postgres-platform#the-free-tier) (15GB of storage, HA included, no pausing) and in general a very good offering for “small scale”. Having this cell-splitting system in a region ensures that one noisy neighbor will not affect the whole region. It will, however, affect the cell they live in. In order to mitigate the noisy neighbor issues, we have rate limits and concurrency limits in place and we are monitoring for abuse.
```